lib/maintenance-window.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+const log = require('bole')('fabrik8.cluster')
+
+exports.setMaintenanceWindow = async function (clusterClient, windows, cluster) {
+  const { environment = 'production', zones = [] } = cluster
+  const clusterOpts = {
+    clusterId: cluster.name,
+    zone: zones[0],
```

Comment:
Let's future proof here and iterate through zones instead of just going to the first one.

Reply:
Actually, we may decide different zones get different windows so maybe we have that discussion later

Reply:
I have no idea what this query would look like with a multi-zone cluster.  Cross that bridge when we get there, I guess.